### ISSUE TITLE
question: can you use multiple colors in a chart?

### ISSUE THREAD

**Issue by beebase** [capitalone/react-native-pathjs-charts#5]
thanks.

**Comment by caledhwa** [comment]
Not yet. This is a good feature... I already need this. Ultimately, this involves overriding the fancy color algorithm I inherited in the port. I'd think you'd want to be able to pull color data from the data and not styling info... Still thinking on a good approach. Ideas?

**Comment by marzolfb** [comment]
Agreed, this is a desired feature. It may be easier to submit several PRs adding multi-color support for a specific chart at a time or a set of charts at a time.

**Comment by SimchaShats** [comment]
I think the good thing alternatively in color attribute pass array of colors, and to check if there only one color then using previous algorithm and if there is in array than just to take colors... It's also backward compatible... And I there are number of colors less then number of data then use algorithm for the last color...

**Comment by marzolfb** [comment]
Multi-color support does exist - see my comments on [this closed PR](https://github.com/capitalone/react-native-pathjs-charts/pull/22)
